addons/stock_dropshipping/stock_dropshipping.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, api, _
+from openerp.exceptions import Warning
+
+class stock_warehouse(models.Model):
+    _inherit = 'stock.warehouse'
+
+    @api.model
+    def _get_drop_shipping_route(self):
+        try:
+            ds_route_id = self.env.ref("stock_dropshipping.route_drop_shipping")
+        except:
+            ds_route_id = self.env['stock.location.route'].search([('name', 'like', _('Drop Shipping'))])
+            ds_route_id = ds_route_id and ds_route_id[0] or False
+        if not ds_route_id:
+            raise Warning(_('Can\'t find any generic Drop Shipping route.'))
```

Comment:
I thought it could be useful when calling function separately. But If you think it is not much useful then I can remove the function "_get_drop_shipping_route" and will do the searching it offers in "_check_routing" itself.